frontend/src/shared/helpers.tsx

```diff
@@ -151,9 +144,9 @@ export const PatientsListColumns: ColDef[] = [
           variant="outline-secondary"
           size="sm"
           onClick={() => {
-            params.context.navigateFunction(
-              `/patients/${params.data.smilePatientId}`
-            );
+            const patientId =
```

Comment:
I'm going to request that we keep the `smilePatientId` as the path value if only to ensure that we're flexible with what kinds of patient aliases we support in the future (i.e., if we start supporting a datasource that has neither a cmo patient id or dmp patient id as the "primary" patient id)

Reply:
That makes sense. I also think that if we have a new patient alias (let's call it "someNewPatientAliasId"), we could keep this `patientId` structure and use "someNewPatientAliasId" as the URL query param. For example:
```ts
// This PR right now
const patientId = params.data.cmoPatientId || params.data.dmpPatientId;

// In the future, when we have a new patient alias
const patientId = params.data.cmoPatientId || params.data.dmpPatientId || params.data.someNewPatientAliasId;
```
This way the URL can be more user-friendly and we can still be flexible. What do you think?

Reply:
✔️  discussed and confirmed that any valid patient ID can be manually entered in the URL and the corresponding patient sample's table will load. Ex: patient with alias C-AAAAA and P-99999 will both render the same table of samples. 

Okay to keep this change as long as we keep in mind that we may need to support additional patient aliases in the future.